modules/ROOT/pages/java-support.adoc

```diff
@@ -91,30 +91,26 @@ If you want to change the cipher's default values, see xref:mule-runtime::tls-co
 [[anypoint-connectors-and-modules]]
 == Anypoint Connectors and Modules
 
-Anypoint Connectors and modules are certified by MuleSoft to run on Java 17. For information about Java 17 support for each connector and module, see the xref:release-notes::connector/anypoint-connector-release-notes.adoc[Release Notes] for that Mule 4 connector or module. Java compatibility information is also provided on the Exchange tiles for connectors and modules. 
+Anypoint Connectors and modules are certified by MuleSoft to run on Java 17. For information about Java 17 support for each connector and module, see the xref:release-notes::connector/anypoint-connector-release-notes.adoc[Release Notes] for that Mule 4 connector or module. 
 
-The most commonly used Anypoint Connectors and modules are Java 17-compatible now, and others are being released regularly. For a list of Java 17-compatible connectors, see https://help.salesforce.com/s/articleView?id=000782248&type=1[Java 17-Compatible Anypoint Connectors^].
+=== Where to Find Java Compatibility Information for Connectors and Modules
 
-To upgrade Anypoint Connectors:
+The most commonly used Anypoint Connectors and modules are Java 17-compatible now, and others are being released regularly. 
 
-. Update your Mule app to run on Mule runtime 4.6 or later. 
-. Change the target JVM at the project property level. This includes both upgrades and downgrades of the JVM version.
-. In Anypoint Studio, update the connectors in your app to the latest Java 17-compatible version. 
-. xref:mule-runtime::deploying.adoc[Deploy your applications] to a target running on Mule 4.6 or later.
+Java compatibility information is provided on the details page for connectors and modules in the *Asset versions* table. This is the most current compatibility information, as connectors are tagged as Java 17-compatible when they are released. 
```

Comment:
> details page for connectors and modules in the *Asset versions* table.

Does this refer to exchange or the release notes? If exchange, then a link and maybe a screenshot would be helpful